src/d3a/d3a_core/sim_results/plotly_graph.py

```diff
@@ -152,6 +152,10 @@ def plot_slider_graph(cls, fig, stats_plot_dir, area_name, market_slot_data_mapp
             for k in range(market_slot_data_mapping[i].start,
                            market_slot_data_mapping[i].end):
                 step["args"][0]["visible"][k] = True  # Toggle i'th trace to "visible"
+            if market_slot_data_mapping[i].start == market_slot_data_mapping[i].end and \
+                    market_slot_data_mapping[i].start > 0 and \
+                    market_slot_data_mapping[i].end < len(fig.data):
```

Comment:
Maybe this will make it a bit more understandable:
```suggestion
            if market_slot_data_mapping[i].start == market_slot_data_mapping[i].end and \
                    0 < market_slot_data_mapping[i].start < len(fig.data):
```